.eslintrc

```diff
@@ -1,20 +0,0 @@
-{
```

Comment:
I like your code a lot, it feels more mature than my original version and it's more similar to how I write these days. I still like to do a hack here and there, but I tend to favor more lines if it helps readability these days more.